torch/autograd/__init__.py

```diff
@@ -29,33 +29,31 @@ def _make_grads(outputs, grads):
             else:
                 new_grads.append(None)
         else:
-            raise TypeError("gradients can be either Tensors, Variables or None, but got " +
+            raise TypeError("gradients can be either Tensors or None, but got " +
                             type(grad).__name__)
     return tuple(new_grads)
 
 
-def backward(variables, grad_variables=None, retain_graph=None, create_graph=False):
-    """Computes the sum of gradients of given variables w.r.t. graph leaves.
+def backward(tensors, grad_tensors=None, retain_graph=None, create_graph=False):
```

Comment:
`grad_variables => grad_tensors` is a breaking change.  Maybe we can have both available and deprecate `grad_variables` first?

Reply:
Yikes... that sounds reasonable to me